sorted_words.rb

```diff
@@ -0,0 +1,32 @@
+# Author:  Irene Sanchez
+# Discription: The program prompts the user for words and add each to an array 
+# until they hit enter to return a blank word. The array is then displayed sorted. 
+
+class SortedWords 
+    attr_accessor :words
+
+    def initialize 
+        @words = []
+    end
+
+    # Method that asks user for words until blank word is entered
+    def get_words
+        word = " "
+        until word == "" do
+            puts "Enter any word (To exit, click enter): "
+            word = gets.chomp
+            @words.push(word)
+        end
+    end
+
+    # to string method
+    def to_s
+        @words.pop
```

Comment:
rather than removing the last element from the array here I would add a line like `break if word == ""` after line 17 and then you avoid getting that last empty string in your array